### PR TITLE
 maui.ms

### PR DESCRIPTION
**Pull request by adambash**

### CODE REVIEW

**Comment by malonnnn** [comment]
remove the last comma in the powers array

File: CommandHelper/LocalPackages/commands/buffs/maui.ms [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+                "/haste 0-8"
+            ),
+            'buffs': array(
+                "/effect ". @plr ." 5 9999 20 true",
+                "/effect ". @plr ." 10 9999 1 true",
+                "/effect ". @plr ." 11 9999 3 true",
+                "/effect ". @plr ." 13 9999 1 true",
+                "/effect ". @plr ." 16 9999 1 true",
+                "/effect ". @plr ." 21 9999 14 true"
+                "/give ". @plr ." minecraft:fishing_rod 1 16:15 name:&3Maui's&1Fishing&4Rod"
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/maui.ms]: Forgot to give underscores between words

File: CommandHelper/LocalPackages/commands/buffs/maui.ms [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+                "/jump 0-8",
+                "/haste 0-8"
+            ),
+            'buffs': array(
+                "/effect ". @plr ." 5 9999 20 true",
+                "/effect ". @plr ." 10 9999 1 true",
+                "/effect ". @plr ." 11 9999 3 true",
+                "/effect ". @plr ." 13 9999 1 true",
+                "/effect ". @plr ." 16 9999 1 true",
+                "/effect ". @plr ." 21 9999 14 true"
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/maui.ms]: missing a comma

**Comment by adambash** [comment]
In The Trailer when he was telling Moana His Life it showed he jumped right next to the sun and with his magic fishing hook he pulled an island out of the sea. That Explains his haste and jump. If You Don't Believe Me Watch The Trailer It Shows Over there. He Also Has ShapeShifting Abilities Cause He Can Turn Into Animals In The Movie And Trailer.

File: CommandHelper/LocalPackages/commands/buffs/maui.ms [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+            'op': 'false',
+            'speech': "Maui, Demigod of the Wind and Sea! ",
+            'powers': array(
+                "/buff",
+                "/flyspeed <number>",
+                "/fly",
+                "/carryplayer",
+                "/dismountplayer",
+                "/run 0-2",
+                "/jump 0-8",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/maui.ms]: you can give him jump 9

File: CommandHelper/LocalPackages/commands/buffs/maui.ms [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+                "/haste 0-8"
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/maui.ms]: you can give him haste 9

File: CommandHelper/LocalPackages/commands/buffs/maui.ms [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+            'val': 'eyJ0aW1lc3RhbXAiOjE0ODE5NTA1MTM5OTcsInByb2ZpbGVJZCI6IjQwODdiYzdmYzIxMzQ3ZGZiZjJkNDE2NTAwMzBiZTQ1IiwicHJvZmlsZU5hbWUiOiJhZGFtYmFzaCIsInNpZ25hdHVyZVJlcXVpcmVkIjp0cnVlLCJ0ZXh0dXJlcyI6eyJTS0lOIjp7InVybCI6Imh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvN2Q4Mjg3Zjk0NmM5YmU5NDhhYjQ1ZTRkNTM1OTNkNmM4ZTcxOGUwZjJjYmU5N2M2NDA4NGQ4MjdjNzc1NDUifX19',
+            'trail': '',
+            'op': 'false',
+            'speech': "Maui, Demigod of the Wind and Sea! ",
+            'powers': array(
+                "/buff",
+                "/flyspeed <number>",
+                "/fly",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/maui.ms]: EVIDENCE REQUIRED: flying

File: CommandHelper/LocalPackages/commands/buffs/maui.ms [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+            'val': 'eyJ0aW1lc3RhbXAiOjE0ODE5NTA1MTM5OTcsInByb2ZpbGVJZCI6IjQwODdiYzdmYzIxMzQ3ZGZiZjJkNDE2NTAwMzBiZTQ1IiwicHJvZmlsZU5hbWUiOiJhZGFtYmFzaCIsInNpZ25hdHVyZVJlcXVpcmVkIjp0cnVlLCJ0ZXh0dXJlcyI6eyJTS0lOIjp7InVybCI6Imh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvN2Q4Mjg3Zjk0NmM5YmU5NDhhYjQ1ZTRkNTM1OTNkNmM4ZTcxOGUwZjJjYmU5N2M2NDA4NGQ4MjdjNzc1NDUifX19',
+            'trail': '',
+            'op': 'false',
+            'speech': "Maui, Demigod of the Wind and Sea! ",
+            'powers': array(
+                "/buff",
+                "/flyspeed <number>",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/maui.ms]: EVIDENCE REQUIRED: super fast flying

File: CommandHelper/LocalPackages/commands/buffs/maui.ms (hunk 5 above)
Comment (adambash) [inline CommandHelper/LocalPackages/commands/buffs/maui.ms]: i gave evidence he can turn into an eagle so he can fly the link is up there

File: CommandHelper/LocalPackages/commands/buffs/maui.ms (hunk 6 above)
Comment (adambash) [inline CommandHelper/LocalPackages/commands/buffs/maui.ms]: eagles are super fast flyers

File: CommandHelper/LocalPackages/commands/buffs/maui.ms [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+                "/fly",
+                "/carryplayer",
+                "/dismountplayer",
+                "/run 0-2",
+                "/jump 0-8",
+                "/haste 0-8"
+                "/disguise",
```
Comment (adambash) [inline CommandHelper/LocalPackages/commands/buffs/maui.ms]: he can turn into a bird

File: CommandHelper/LocalPackages/commands/buffs/maui.ms [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+                "/disguiseviewself",
+                "/undisguise"
+            ),
+            'buffs': array(
+                "/effect ". @plr ." 5 9999 20 true",
+                "/effect ". @plr ." 10 9999 1 true",
+                "/effect ". @plr ." 11 9999 3 true",
+                "/effect ". @plr ." 13 9999 1 true",
+                "/effect ". @plr ." 16 9999 1 true",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/maui.ms]: EVIDENCE REQUIRED: night vision
the trailer did not show this

File: CommandHelper/LocalPackages/commands/buffs/maui.ms [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+                "/haste 0-8"
+                "/disguise",
+                "/disguiseviewself",
+                "/undisguise"
+            ),
+            'buffs': array(
+                "/effect ". @plr ." 5 9999 20 true",
+                "/effect ". @plr ." 10 9999 1 true",
+                "/effect ". @plr ." 11 9999 3 true",
+                "/effect ". @plr ." 13 9999 1 true",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/maui.ms]: EVIDENCE REQUIRED: night vision
the trailer did not show this

File: CommandHelper/LocalPackages/commands/buffs/maui.ms [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+                "/run 0-2",
+                "/jump 0-8",
+                "/haste 0-8"
+                "/disguise",
+                "/disguiseviewself",
+                "/undisguise"
+            ),
+            'buffs': array(
+                "/effect ". @plr ." 5 9999 20 true",
+                "/effect ". @plr ." 10 9999 1 true",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/maui.ms]: EVIDENCE REQUIRED: regeneration
the trailer did not show this

File: CommandHelper/LocalPackages/commands/buffs/maui.ms (hunk 6 above)
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/maui.ms]: a bald eagle can fly up to 100 miles per hour.
thats not very fast.

File: CommandHelper/LocalPackages/commands/buffs/maui.ms (hunk 5 above)
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/maui.ms]: at what time in the trailer does he turn into an eagle? i did not see it

File: CommandHelper/LocalPackages/commands/buffs/maui.ms (hunk 9 above)
Comment (adambash) [inline CommandHelper/LocalPackages/commands/buffs/maui.ms]: DEMIGOD OF THE SEA.

File: CommandHelper/LocalPackages/commands/buffs/maui.ms (hunk 10 above)
Comment (adambash) [inline CommandHelper/LocalPackages/commands/buffs/maui.ms]: i'll take that away

File: CommandHelper/LocalPackages/commands/buffs/maui.ms (hunk 8 above)
Comment (adambash) [inline CommandHelper/LocalPackages/commands/buffs/maui.ms]: i'll take that away

**Pull request merged**